docs/source/elastalert.rst

```diff
@@ -111,6 +111,9 @@ even if ElastAlert is stopped and restarted, it will never miss data or look at
 files in this folder that end in .yaml. If the contents of this folder change, ElastAlert will load, reload
 or remove rules based on their respective config files.
 
+``scan_subdirectories``: If true, ElastAlert will recursively scan ``rules_folder`` for yaml files to load as rules.
```

Comment:
I feel like `scan_subdirectories: true` should just be the behavior and we could remove the option.
Sometimes there's a LOT of options.